### PR TITLE
added new theme: "Documentation theme for Jekyll"

### PR DESCRIPTION
**Pull request by tomjohnson1492**
I added a new theme here following the instructions you provided. This theme is designed for technical writers doing single sourcing with documentation projects.

### CODE REVIEW

File: _posts/2015-12-13-documentation-theme-jekyll.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Documentation Theme for Jekyll
+date: 2015-12-13 12:55
+homepage: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/home.html
+download: https://github.com/tomjohnson1492/documentation-theme-jekyll/archive/gh-pages.zip
+demo: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/home.html
+author: tomjohnson1492
+thumbnail: docthemejekyll.png
```
Comment (gynter) [inline _posts/2015-12-13-documentation-theme-jekyll.markdown]: Hello and thanks for the PR! Please match the thumbnail name with post name: `documentation-theme-jekyll.png`. Thanks!

**Comment by tomjohnson1492** [comment]
I updated the thumbnail name file. It should be good to merge. Thanks.

**Comment by gynter** [comment]
It still shows me image dimensions as 252px × 202px. Please update image to 250px × 200px.

**Comment by tomjohnson1492** [comment]
Sorry about that. I fixed the thumbnail size now.

**Comment by tomjohnson1492** [comment]
Is it ready to merge now?

**Comment by gynter** [comment]
Yep nice, thanks!

**Pull request merged**